app/javascript/src/components/Common/SideMenu/index.jsx

```diff
@@ -7,7 +7,7 @@ import PropTypes from "prop-types";
 
 const SideMenu = ({ title, items, showMenu }) => (
   <MenuBar showMenu={showMenu} title={title}>
-    {items.main.map((item, index) => (
+    {(items.main ?? []).map((item, index) => (
```

Comment:
Why do we need a null check here?

Reply:
In case the `items.main` is undefined/null. It will not happen with the current demo data provided, as all data SideMenu items have a `main` key. Added this as I was testing it with incomplete data.